src/renderer/components/watch-video-info/watch-video-info.scss

```
@@ -22,6 +22,17 @@
   gap: 10px;
 }
 
+.unlistedBadge {
+  background-color: var(--secondary-card-bg-color);
+  border-radius: 5px;
+  display: inline;
+  padding-inline: 5px;
+  white-space: nowrap;
+  font-size: 12.8px;
+  line-height: 8px;
```

```suggestion
  font-size: 13px;
```
Unless there is an important reason I don't see why use fractional value
See https://stackoverflow.com/questions/7656883/using-fractional-ems-in-csss-font-size-property
`line-height` makes no effect (I toggle it in developer tool)